build/style.css

```diff
@@ -240,35 +210,70 @@ h2:hover a .anchor-hint {
     margin-bottom: 0.2rem;
 }
 
-.view-overview .details,
-.view-overview .block-image {
+.view-summary .details,
+.view-summary .block-image {
     display: none;
 }
 
-.view-overview .headline {
+.view-summary .headline {
     max-width: none;
     margin-bottom: 0;
 }
 
-.view-overview .blocks {
+.view-summary .blocks {
     display: grid;
     grid-template-columns: 1fr 1fr;
     gap: 0.5rem;
     margin: 0 0.5rem;
 }
 
-.view-overview .blocks .block {
+.view-summary .blocks .block {
     margin: 0;
 }
 
-.view-overview .blocks .block:last-child {
+.view-summary .blocks .block:last-child {
     grid-column: 1 / span 2;
 }
 
+.utility-bar {
+    display: flex;
+    justify-content: flex-end;
+    flex-wrap: wrap;
+    gap: 1rem;
+}
+
+.summary-toggle {
+    width: 1.5rem;
+    height: 1.5rem;
+    margin-top: 0.25rem;
+    opacity: 0.5;
+    background: none;
+    padding: 0 !important;
+    font-size: inherit;
+    font-family: inherit;
+    cursor: pointer;
+    color: var(--color-text-light);
+    border: 2px solid var(--color-text-light);
+    border-radius: 8px;
+    text-align: center;
+}
+
+.summary-toggle:hover {
+    background: var(--color-orange);
+    opacity: 1;
+}
+
+.summary-toggle:before {
+    display: block;
+    content: '-';
```

Comment:
While I get the idea, I'm not sure if this is really clear enough. It also doesn't help that the blocks you're collapsing are off-screen when you click this button, so it's never immediately clear what it does (although the `title` hover text does help). Maybe we could use icons for that instead? Like this for collapse:

![image](https://github.com/infi-nl/the-infi-way/assets/5592716/7205b31b-a650-487b-841a-2bce6415b6d9)

(This specific one I just found after a quick image search, so we'd have to look for something we can use)

Reply:
In my opinion, using up & down chevrons make sense when using them inside of the sections (which was the current/previous situation). Now that the toggle button has moved to the top bar, I fear users will see it as a dropdown menu button. The way we are using this button to globally toggle the content in all sections simultaneously is fairly uncommon and I actually haven't a clue what icon would be best for that particular situation. Perhaps a user test and/or a designer would help us with that, although that would perhaps be overkill right now. 😅 I'll think about it a bit more.

Reply:
What would you think about using a grid / list icons similar to this?

![image](https://github.com/infi-nl/the-infi-way/assets/8208970/d6965d14-89af-4794-8b7f-33fdc433859f)

Reply:
Ohh yeah that would be perfect I think! Good idea.